### PR TITLE
[14910] 2.7.0

### PR DESCRIPTION
**Pull request by MiguelBarro**
This PR bumps the version to 2.7.0 and adds the release notes for v2.7.0

### CODE REVIEW

**Review by jsantiago-eProsima — DISMISSED** [review]
Small nitpick

File: docs/03-exports/aliases-api.include [hunk 1]
```diff
@@ -131,14 +136,15 @@
 .. |DomainParticipant::get_qos-api| replace:: :cpp:func:`get_qos()<eprosima::fastdds::dds::DomainParticipant::get_qos>`
 .. |DomainParticipant::get_publisher_qos_from_profile| replace:: :cpp::func`get_publisher_qos_from_profile()<eprosima::fastdds::dds::DomainParticipant::get_publisher_qos_from_profile>`
 .. |DomainParticipant::get_subscriber_qos_from_profile| replace:: :cpp::func`get_subscriber_qos_from_profile()<eprosima::fastdds::dds::DomainParticipant::get_subscriber_qos_from_profile>`
```
Comment (JLBuenoLopez-eProsima) [inline docs/03-exports/aliases-api.include:138]: This other two aliases are missing the `:` too. I suggest fixing it in this PR.

File: docs/notes/notes.rst [hunk 2]
```diff
@@ -3,25 +3,26 @@
...
-4. Support for :cpp:func:`DataReader::lookup_instance()<eprosima::fastdds::dds::DataReader::lookup_instance>`
-5. Support for SampleLostStatus on DataReader
-6. Improved doxygen documentation
+1. Support of :ref:`DDS SampleRejectedStatus API <dds_layer_core_status_sampleRejectedStatus>`
+2. Support of DDS DataWriter methods:
+   + `DataWriter::write_w_timestamp()<eprosima::fastdds::dds::DataWriter::write_w_timestamp>`
+   + `DataWriter::register_instance_w_timestamp()<eprosima::fastdds::dds::DataWriter::register_instance_w_timestamp>`
+   + `DataWriter::unregister_instance_w_timestamp()<eprosima::fastdds::dds::DataWriter::unregister_instance_w_timestamp>`
+   + `DataWriter::dispose_w_timestamp()<eprosima::fastdds::dds::DataWriter::dispose_w_timestamp>`
+3. Support of DDS `DomainParticipant::find_topic()<eprosima::fastdds::dds::DomainParticipant::find_topic>`
```
[the remark this answers is not in the document]
Comment (MiguelBarro) [reply docs/notes/notes.rst]: ack

**Review by JLBuenoLopez-eProsima — DISMISSED** [review]
A couple of nitpicks though the PR looks also good without them!

File: docs/03-exports/aliases-api.include (hunk 1 above)
Comment (MiguelBarro) [reply docs/03-exports/aliases-api.include:138]: ack

File: docs/notes/notes.rst [hunk 3]
```diff
@@ -3,25 +3,26 @@
...
-3. Support for :cpp:func:`DataWriter::get_key_value()<eprosima::fastdds::dds::DataWriter::get_key_value>`
-4. Support for :cpp:func:`DataReader::lookup_instance()<eprosima::fastdds::dds::DataReader::lookup_instance>`
-5. Support for SampleLostStatus on DataReader
-6. Improved doxygen documentation
+1. Support for :ref:`DDS SampleRejectedStatus API <dds_layer_core_status_sampleRejectedStatus>`
+2. Support for DDS DataWriter methods:
+   + `DataWriter::write_w_timestamp()<eprosima::fastdds::dds::DataWriter::write_w_timestamp>`
+   + `DataWriter::register_instance_w_timestamp()<eprosima::fastdds::dds::DataWriter::register_instance_w_timestamp>`
+   + `DataWriter::unregister_instance_w_timestamp()<eprosima::fastdds::dds::DataWriter::unregister_instance_w_timestamp>`
+   + `DataWriter::dispose_w_timestamp()<eprosima::fastdds::dds::DataWriter::dispose_w_timestamp>`
```
[the remark this answers is not in the document]
Comment (MiguelBarro) [reply docs/notes/notes.rst]: ack

**Review by MiguelCompany — DISMISSED** [review]
LGTM

**Review by MiguelCompany — APPROVED** [review]
LGTM with green CI

**Pull request merged**